requirements_distributed.txt

```diff
@@ -3,7 +3,7 @@ dask[dataframe]
 pyarrow==6.0.1 # https://github.com/ray-project/ray/issues/22310
 
 # requirements for horovod
-horovod[pytorch]>=0.24.0
+horovod[pytorch]>=0.24.0,<0.26.0
```

Comment:
You shouldn't need to pin horovod if you have the packaging requirement.

Reply:
I tried adding packaging just before the horovod install but it still caused the same error [add packaging to distributed](https://github.com/ludwig-ai/ludwig/pull/2642/commits/6c910c1e19aa2d7e7da5ad2e88ccfe61c5cede1a)

Reply:
Got it! Let me add that in

Reply:
Didn't actually know we could do `!=` in requirements.txt to make sure a certain version was never installed